gitea/repo_key.go

```diff
@@ -13,10 +13,10 @@ import (
 
 // DeployKey a deploy key
 type DeployKey struct {
-	ID       int64     `json:"id"`
-	Key      string    `json:"key"`
-	URL      string    `json:"url"`
-	Title    string    `json:"title"`
+	ID    int64  `json:"id"`
```

Comment:
This change does not seems to be related to this PR as only whitespace are changed here

Reply:
Yeah, I just run `fmt` and I committed the changes.
Those changes need to be done in another PR?

Reply:
I think yes